### PR TITLE
Update to 0.215.0

### LINKED ISSUE #26: Update to the new CLI
### Comment:

image

### PR DESCRIPTION
**Pull request by dhirschfeld**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

----

Resolves #26

### CODE REVIEW

**Comment by dhirschfeld** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,76 +1,47 @@
...
 
 about:
-  home: https://github.com/databricks/databricks-cli
-  license: Apache-2.0
-  license_family: APACHE
-  license_file: LICENSE
   summary: A command line interface for Databricks
-  dev_url: https://github.com/databricks/databricks-cli
+  home: https://github.com/databricks/cli
+  license: LicenseRef-Databricks-License
```
Comment (dhirschfeld) [inline recipe/meta.yaml:36]: * https://github.com/databricks/cli?tab=License-1-ov-file
* https://www.databricks.com/legal/db-license

**Comment by dhirschfeld** [comment]
Maybe not faster, but it does clear a lot of cruft from the build logs.

**Comment by dhirschfeld** [comment]
```
(%PREFIX%) %SRC_DIR%>databricks --help 
'databricks' is not recognized as an internal or external command,
operable program or batch file.
```

Might be that `$PREFIX/bin` isn't on the PATH and need `LIBRARY_BIN` on Windows 🤔

**Comment by dhirschfeld** [comment]
Will have to wait for tomorrow, unless anyone else wants to pick it up...

**Comment by dhirschfeld** [comment]
ping @conda-forge/databricks-cli this should be good to merge.

**Comment by thewchan** [comment]
@andrewmchen @jaroslawk @oblute what do you all think?

**Comment by dhirschfeld** [comment]
No objections?

**Comment by thewchan** [comment]
@dhirschfeld let's do it then

**Comment by thewchan** [comment]
@conda-forge-admin, please rerender

**Pull request merged**